portality/tasks/harvester_helpers/epmc/client.py

```diff
@@ -58,7 +60,16 @@ def __init__(self, message, rawstring, *args, **kwargs):
 
 
 class EuropePMC(object):
-    job = None
+
+    logger = None
+
+    @classmethod
+    def _write_to_logger(cls, msg):
+        try:
+            cls.logger.write(msg)
+        except:
+            app.logger.warn("Logger not provided for Harvester Background task, message: {}").format(msg)
```

Comment:
Probably no need to issue the warning here, just write to the `app.logger` if no custom logger is supplied